cloudinit/apport.py

```diff
@@ -71,6 +72,13 @@
 CLOUDINIT_OUTPUT_LOG = "/var/log/cloud-init-output.log"
 
 
+def is_cloud_init_enabled() -> bool:
+    return (
+        get_status_details(read_cfg_paths()).boot_status_code
+        not in DISABLED_BOOT_CODES
+    )
```

Comment:
Given that this is more status-related functionality than apport-related It is worth just defining this helper function in cloudinit.status?

Reply:
That works for me!

Reply:
Aside: It does feel funny to have code under `cloudinit/cmd/` that ends up getting imported by external tools.

I would almost lean towards a `cloudinit/lib/` to have documented acceptable interfaces for external users with a very small subset of functions that we guarantee to have a stable interface. But that's very much a "someday" idea - for now I think that your suggested approach is best. Thanks for the review @blackboxsw!